platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/Network.java

```diff
@@ -57,6 +59,38 @@ default List<Node> nodes() {
     @NonNull
     Topology topology();
 
+    /**
+     * Configures the network to use a mesh topology with the specified configuration.
+     *
+     * <p>This method must be called before any nodes are added to the network. It configures the topology with the
+     * specified latency, jitter, and bandwidth settings that apply to all node-to-node connections.
+     *
+     * <p>If this method is not called, the default mesh topology configuration is used.
+     *
+     * @param configuration the mesh topology configuration to apply
+     * @return this {@code Network} instance for method chaining
+     * @throws NullPointerException if {@code configuration} is {@code null}
+     * @throws IllegalStateException if any nodes have already been added to the network
+     */
+    @NonNull
+    Network withMeshTopology(@NonNull MeshTopologyConfiguration configuration);
```

Comment:
There should be only one method to configure the topology. We plan to add more topology types later, and it would be excessive to have a method for each.